### PR TITLE
Created Image component

### PR DESCRIPTION
**Pull request by anushasunil**
https://user-images.githubusercontent.com/90134948/152678407-85968208-8afa-442e-9260-e48e898fc22f.mp4

### CODE REVIEW

**Review by aishwarya-bm — APPROVED** [review]
looks all good

File: Stylesheets/Components/images.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+}
+
+.image-container, .image-container img{
+    margin:0 auto;
+}
+
+.image-container.circle {
+    border-radius: var(--circle-border);
+    overflow: hidden;
+}
```
Comment (vivekmehto) [inline Stylesheets/Components/images.css:12]: add square image class.

**Review by vivekmehto — COMMENTED** [review]
looks good just one suggestion you can give square class.

**Review by ttahm3d — APPROVED** [review]
Everything looks good. Add Squre images and you can merge

File: Stylesheets/Components/images.css (hunk 1 above)
Comment (anushasunil) [reply Stylesheets/Components/images.css:12]: Okay, I have done the changes, there's no change in the preview. Please check it once.

**Review by vivekmehto — APPROVED** [review]
Looks Awesome.

**Pull request merged**